classes/OccurrenceAttributeSearch.php

```diff
@@ -75,18 +75,18 @@ private function getTraitSearchHTML($traitID,$classStr=''){
 					continue;
 				}
 				else{
-					$retStr .= '<div title="'.$sArr['description'].'" style="clear:both">';
-					$retStr .= '<input name="attr[]" id="traitstateid-'.$sid.'" class="'.$classStr.'" type="checkbox" value="'.$sid.'" onchange="traitChanged(this)" /> ';
-					$retStr .= $sArr['name'];
+					$retStr .= '<div title="' . $sArr['description'] . '" style="clear:both;">';
+					$retStr .= '<input data-chip="' . $this->traitArr[$traitID]['name'] . ': ' . ucfirst($sArr['name']) . '" name="attr[]" id="traitstateid-' . $sid . '" class="' . $classStr . '" type="checkbox" value="' . $sid . '" onchange="traitChanged(this)" /> ';
```

Comment:
Adds a data-chip attribute to this. Essential for the new search page and harmless in harvestparams.php, which will share this code.